devtools/ci/jenkins/Dockerfile.libcpptraj

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda:latest
```

Comment:
should we make a base image (saying: `ambermd/libcpptraj-base`) using line 1 to 14 first (that is built based on continuumio/miniconda:latest but do all `apt-get` and 
```
                sh "conda install -y --file conda-requirements.txt"
                sh "pip install -r pip-requirements.txt"
```

Then

```
FROM ambermd/libcpptraj-base
COPY . /cpptraj

 ENV CPPTRAJHOME "/cpptraj"

 RUN cd /cpptraj && \
    ./configure -openmp -shared gnu && \
    make -j4 libcpptraj
```

So we don't need lie 27-2 in this file: https://github.com/Amber-MD/pytraj/pull/1482/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R27

The aim is that we only let pytraj build fail because of cpptraj itself.

Reply:
(Or I just overthink)

Reply:
Sounds like a bigger maintenance headache to me than anything else.  This image serves *one* purpose, which is providing a pre-packaged cpptraj install set up specifically to support testing pytraj against a cpptraj commit inside Jenkins.

That's why I'm comfortable doing a whole bunch of `chmod`s and leaving a suboptimal/dirty environment in place.  I don't expect this image to actually be *used* for anything else.  Nor do I think the `ambermd/libcpptraj` image will be of general interest.

If the base you propose making turns out to be generally helpful to more than just this image, I'd support making it.  But until then, I'd rather not have to maintain *both* `ambermd/libcpptraj-base` *and* this one.  But I don't really fancy the idea of maintaining 2 images for only this use-case.

Besides, the current `Dockerfile.libcpptraj` is only 22 lines.  Yours is 8.  Seems like a lot of work to save 14 lines of trivially understandable `Dockerfile`.